htdocs/core/boxes/box_clients.php

```diff
@@ -140,7 +140,7 @@ function loadBox($max=5)
                     );
 
                     $this->info_box_contents[$line][] = array(
-                        'td' => 'align="right" width="18"',
+                        'td' => 'align="right" width="18" class="classfortooltip"',
```

Comment:
this class allow to add content of a <a href title into a popup.
But here we are on a td html tag. Are you sure of this change ?